packages/go/test/fixtures/21-square-brackets-with-go-mod/probes.json

```diff
@@ -0,0 +1,9 @@
+{
+  "probes": [
+    {
+      "path": "/api/[hello].go",
+      "mustContain": "hello:RANDOMNESS_PLACEHOLDER"
+    },
+    { "path": "/api/sub/[hi].go", "mustContain": "hi:RANDOMNESS_PLACEHOLDER" }
```

Comment:
Is this correct for zero config? I wouldn’t expect the file extension in the path name. And the brackets should be removed.